proto/pairing/query.proto

```diff
@@ -133,6 +133,8 @@ message QueryVerifyPairingRequest {
 message QueryVerifyPairingResponse {
   bool valid = 1;
   int64 index = 2;
+  uint64 pairedProviders = 3;
```

Comment:
nit: isn't the coding style in *.proto files like "paired_providers"?
and also "cu_per_epoch" below?